.maestro/release_tests/content-blocking.yaml

```diff
@@ -49,4 +49,9 @@ tags:
 
 - tapOn: 
     id: PrivacyIcon
-- assertVisible: "Protections are OFF for this site"
+- extendedWaitUntil: 
+    visible: "Protections are OFF for this site"
```

Comment:
@brindy I'm not sure this fixes the issue, even though I do agree it's a timing issue. 
From what I see, switching protections on and off triggers a page refresh "after a while". If this refresh happens when the dashboard is open, the dashboard is dismissed. This refresh seems to happen even if you trigger a manual refresh, as we do in the test. 
I can reproduce this even manually if I'm fast enough with all the steps. 
So, I _think_ we should wait for the automatic refresh to happen _before_ opening the privacy dashboard. 
Does it make sense to you?

Reply:
That sounds like a dashboard bug then right?

Reply:
Is it a bug? I'd say it's something we could improve, e.g. by triggering the automatic refresh faster or not triggering if a manual refresh has happened in the meantime. Or by pushing an updated state to the dashboard after a refresh instead of dismissing it. But I don't know this is a big problem for users in every day usage and I don't know if any of these things are relatively easy to do. 

But, you are also more familiar than me with the expected behaviour of the dashboard, so if you may see a bug in this behaviour that I don't.

Reply:
Yeah, I'd say this is a bug. I think it's intentional to close it, but I don't think the user should be able to open it and have it re-close on them.  I will create a task (but I think it's on @jaceklyp  who is out for a bit).  So I will add a temporary work around here by adding a delay if I can and will add to the task to adjust the UI test.